BridgeAppSDK/SBAJSONObject.m

```diff
@@ -34,8 +34,24 @@
 #import "SBAJSONObject.h"
 @import BridgeSDK;
 
+@implementation NSObject (SBAJSONObject)
```

Comment:
Not really a huge fan of extending NSObject with an NSAssert. What is the intended use-case? Additionally, this could return `[self description]` rather than asserting or returning an empty dictionary.